src/templates/button-back.jsx

```diff
@@ -25,14 +25,14 @@ export const BackButton = ({ global, isEnabled, translate, unselectNote }) => {
     });
 
     return isEnabled
-        ? <a className="wpnc__back" onClick={routeBack(global, unselectNote)}>
+        ? <button className="wpnc__back" onClick={routeBack(global, unselectNote)}>
               <Gridicon icon="arrow-left" size={18} />
               {backText}
-          </a>
-        : <a className="wpnc__back disabled" disabled="disabled">
+          </button>
+        : <button className="wpnc__back disabled" disabled="disabled">
```

Comment:
This can be `<button className="wpnc__back disabled" disabled>`. React is also smart enough to handle html5 boolean props correctly:

So this might change to:
```jsx
<button 
     className="wpnc__back" 
     onClick={routeBack(global, unselectNote)}
      disabled={!isEnabled}
>
```